### ISSUE TITLE
Remove OldSignal module

### ISSUE THREAD

**Issue by 1ForeverHD** [1ForeverHD/ZonePlus#52]
This should have been removed in the previous release

**Comment by 1ForeverHD** [comment]
Yep, I'm planning to remove this next week but you're welcome to submit a PR to development to fix this now 👍

**Comment by VSCPlays** [comment]
Did you remove it now? it's been like almost a year as you left the issue open
